modules/library/jdbc/src/main/java/org/geotools/jdbc/JDBCDataStore.java

```diff
@@ -418,6 +424,14 @@ public void setFetchSize(int fetchSize) {
         this.fetchSize = fetchSize;
     }
 
+    public int getBatchInsertSize() {
```

Comment:
I refuse to add stupid comments like: "get the batch insert size" and the attribute itself has enough documentation.